apps/frontend/src/App.css

```
@@ -48,6 +48,16 @@ body {
   background: #6b7280;
 }
 
+/* Mobile drawer slide-in animation */
+`@keyframes` slide-in {
```

Stylelint (17.6.0)

[error] Unexpected unknown type selector "`@keyframes`" (selector-type-no-unknown)

(selector-type-no-unknown)